src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs

```diff
@@ -26,6 +27,7 @@ public class FillReferenceInformation : BaseDocumentBuildStep, ICanTraceContextI
 
         public override void Postbuild(ImmutableList<FileModel> models, IHostService host)
```

Comment:
@herohua

---
In reply to: [107823555](https://github.com/dotnet/docfx/pull/1466#discussion_r107823555) [](ancestors = 107823555)